contractcourt/htlc_timeout_resolver.go

```diff
@@ -104,6 +104,11 @@ const (
 	// swept on-chain by them with pre-image.
 	expectedRemoteWitnessSuccessSize = 5
 
+	// expectedLocalWitnessSuccessSize is the expected size of the witness
+	// on the local commitment transaction for an outgoing HTLC that is
+	// swept on-chain by us with pre-image.
```

Comment:
If it is the _local_ commitment tx, and we are sweeping the pre-image path, then it must be an incoming HTLC. An _outgoing_ htlc on our commitment tx can only be swept by us via the timeout path.

Reply:
updated.